applications/eups-distributor/values-roundtable-prod.yaml

```diff
@@ -3,3 +3,5 @@ image:
 ingress:
   # -- Additional annotations for the ingress rule
   hostname: "eups.lsst.cloud"
+config:
+  serviceAccountReadOnly: eups-distributor@roundtable-abe2.iam.gserviceaccount.com
```

Comment:
Set to `eups-distributor@roundtable-prod-f6fd.iam.gserviceaccount.com`.   You can find names under `Service Accounts` in [Console](https://console.cloud.google.com/iam-admin/serviceaccounts?cloudshell=true&inv=1&invt=Abx6iA&project=roundtable-prod-f6fd).